configs/default/DIAT-MAMBAF722_2022B.config

```diff
@@ -111,7 +111,8 @@ feature OSD
 
 # serial
 serial 0 64 115200 57600 0 115200
-serial 3 1 115200 57600 0 115200
+serial 2 1 115200 57600 0 115200
+serial 3 0 115200 57600 0 115200
```

Comment:
Remove this line as this is default.

Reply:
which line? you mean 115 (`serial 3 0 115200 57600 0 115200`)?

I did that because I was under the impression that this config is layered on top of the base config

https://github.com/betaflight/unified-targets/blob/master/configs/default/DIAT-MAMBAF722.config

and there it gets enabled

https://github.com/betaflight/unified-targets/blob/6a788a3aa76a2daab7241e5b9090b724a75d316e/configs/default/DIAT-MAMBAF722.config#L111

As far as I understand, without line 115 I would have 2 ports (`serial 2` and `serial 3`) enabled, which would be wrong. Am I mistaken here?

Reply:
`DIAT-MAMBAF722` is not a base target for the other ones.
Line 115 is not needed anymore.

Reply:
done. thanks for the clarification.